maat-court-data-api/src/test/java/gov/uk/courtdata/integration/assessment/FinancialAssessmentControllerIntegrationTest.java

```diff
@@ -454,4 +457,15 @@ private void assertFinancialAssessmentEqual(FinancialAssessmentDTO expectedAsses
         assertThat(expectedAssessment.getInitApplicationEmploymentStatus()).isEqualTo(actualAssessment.getInitApplicationEmploymentStatus());
         assertThat(expectedAssessment.getUpdated()).isEqualTo(actualAssessment.getUpdated());
     }
+
+    @Test
+    @Transactional
+    public void givenAValidRepOrdersAvailable_whenUpdateAppDateCompletedInvoked_theCompletedDateShouldUpdate() {
```

Comment:
Since this is an integration test, what we're really looking for is to ensure that everything works together correctly. Therefore, I'd expect you to be making a mock request to the new endpoint and ensuring the Rep Order gets updated as a result.